deploy/kubernetes/base/controller/controller.yaml

```diff
@@ -23,19 +23,19 @@ spec:
       priorityClassName: csi-gce-pd-controller
       containers:
         - name: csi-provisioner
-          image: gke.gcr.io/csi-provisioner
+          image: k8s.gcr.io/sig-storage/csi-provisioner
           args:
             - "--v=5"
             - "--csi-address=/csi/csi.sock"
             - "--feature-gates=Topology=true"
             - "--metrics-address=:22011"
-            - "--enable-leader-election"
-            - "--leader-election-type=leases"
             - "--leader-election-namespace=$(PDCSI_NAMESPACE)"
             - "--timeout=250s"
             - "--extra-create-metadata"
-          # - "--run-controller-service=false" # disable the controller service of the CSI driver
-          # - "--run-node-service=false"       # disable the node service of the CSI driver
+          # - "--run-controller-service=false"  # disable the controller service of the CSI driver
+          # - "--run-node-service=false"        # disable the node service of the CSI driver
+            - "--leader-election"
+            - "--default-fstype=ext4"
```

Comment:
do we need to set default fstype?
If the reason we need to set it due to issue related to fsgroup will be ignored if default fstype is not set, https://github.com/kubernetes-sigs/vsphere-csi-driver/issues/370#issuecomment-723329053

This logic should be fixed in 1.19 by https://github.com/kubernetes/kubernetes/pull/92001. This means for testing k/k 1.19 and above, no need to set default-fstype here. During format and mount, the default fstype will be applied depending on OS. 

So for OSS test which using the latest kubernetes, we don't need to set default value here so we can test the behavior of correctly setting fstype by driver.

For internal tests which tests earlier version of kubernetes, it  also seems ok so far without setting default-fstype. From what I searched, we didn't test fsgroup behavior yet.

There is a new test added very recently for 1.20. https://github.com/kubernetes/kubernetes/blame/master/test/e2e/storage/testsuites/fsgroupchangepolicy.go

So I think from testing point, should be ok without setting default value here.

Reply:
I agree we don't need it staring with Kubernetes 1.20. In 1.19 it's still an alpha feature and not enabled by default.

I think for the purposes of this PR, which is to cleanup the overlays for the already released driver, we should still keep it.

For the next driver release, we can look into removing it along with adding the CSIDriver change. It will also bump up the min version of this driver to Kubernetes 1.20

Reply:
Opened https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/issues/668 to track this since this PR has merged.